samples/dnn/inpainting.cpp

```diff
@@ -0,0 +1,226 @@
+/*
```

Comment:
OpenCV license should be inserted before BigVision copyright in order to avoid any confusion